fuzz/README.md

```diff
@@ -0,0 +1,88 @@
+# Fuzzing
+
+`hex-conservative` has fuzzing harnesses setup for use with honggfuzz.
+
+To run the fuzz-tests as in CI -- briefly fuzzing every target -- simply
+run
+
+    ./fuzz.sh
+
+in this directory.
+
+To build honggfuzz, you must have libunwind on your system, as well as
+libopcodes and libbfd from binutils **2.38** on your system. The most
+recently-released binutils 2.39 has changed their API in a breaking way.
+
+On Nix, you can obtain these libraries by running
+
+    nix-shell -p libopcodes_2_38 -p libunwind
+
+and then run fuzz.sh as above.
+
+# Long-term fuzzing
+
+To see the full list of targets, the most straightforward way is to run
+
+    source ./fuzz-util.sh
+    listTargetNames
+
+To run each of them for an hour, run
+
+    ./cycle.sh
+
+To run a single fuzztest indefinitely, run
+
+    HFUZZ_BUILD_ARGS='--features honggfuzz_fuzz' cargo hfuzz run <target>
```

Comment:
This could be my fault, its the old way we did it with an explicit feature, not sure how this got back in here, I've remove it before - will investigate.

Reply:
oh woops, I didn't realize this was docs only when I read your comment.